internal/prefixcollector/collector.go

```diff
@@ -71,20 +71,20 @@ func NewExcludePrefixCollector(outputFilePath string, notify *sync.Cond,
 // Start - begin monitoring sources.
 // Updates exclude prefix file after every notification.
 func (epc *ExcludePrefixCollector) Start() {
+	go epc.waitForContextDone()
```

Comment:
Can we replace this goroutine to inline?

Reply:
Replaced.